egs/swbd/s5c/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
+
+. cmd.sh
+
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+train_stage=-10
+has_fisher=true
+speed_perturb=true
+dir=exp/nnet3/nnet_tdnn_a
+train_set=train_nodup
+ali_dir=exp/tri4_ali_nodup
+if [ "$speed_perturb" == "true" ]; then
+  dir=${dir}_sp
+  train_set=train_nodup_sp
+  ali_dir=exp/tri4_ali_nodup_sp
+fi
+
+
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+local/nnet3/run_ivector_common.sh --stage $stage || exit 1;
+if [ $stage -le 8 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+    utils/create_split_dir.pl \
+     /export/b0{3,4,5,6}/$USER/kaldi-data/egs/wsj-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
+  fi
+
+  steps/nnet3/train_tdnn.sh --stage $train_stage \
+    --num-epochs 2 --num-jobs-initial 3 --num-jobs-final 16 \
+    --splice-indexes "-2,-1,0,1,2 -1,0,1,2 -3,-2,-1,0,1,2,3 -7,-6,-5,-4,-3,-2,-1,0,1,2" \
+    --feat-type raw \
+    --online-ivector-dir exp/nnet3/ivectors_${train_set} \
+    --cmvn-opts "--norm-means=false --norm-vars=false" \
+    --initial-effective-lrate 0.0017 --final-effective-lrate 0.00017 \
+    --cmd "$decode_cmd" \
+    --pnorm-input-dim 2750 \
```

Comment:
Please use relu units in place of the pnorm units. 
I would suggest 1024 units.